tenable/sc/__init__.py

```diff
@@ -65,6 +66,7 @@
 from .policies import ScanPolicyAPI
 from .queries import QueryAPI
 from .recast_risks import RecastRiskAPI
+from .reports_definition import ReportsDefinitionAPI
```

Comment:
Should be using a more correct name that conforms to the rest of the package

```suggestion
from .report_definitions import ReportDefinitionsAPI
```